hack/bin/filter-old-releases.jq

```diff
@@ -0,0 +1,9 @@
+.[]
+  | .parsedUpdatedTime = ( .updated
+                         | sub("(?<time>.*)\\.[\\d]+(?<tz>.*)"; "\(.time)\(.tz)")
+                         | strptime("%Y-%m-%d %H:%M:%S %z %Z")
+                         | mktime
+                         )
+  | select ( .parsedTime < (now - 3 * 60 * 60))
+  | [.name, .namespace]
+  | @tsv
```

Comment:
question: why tabs?

Reply:
I think jq can either output a json array, a csv or a tsv. Out of these I thought tsv was the most compatible with bash as I can `awk '{print $1}'`. Do you have a better idea?

Reply:
I just stumbled upon it and was curious, as `FS` usually defaults to *space*.